### PR TITLE
[WIP] Refactor store

### PR DESCRIPTION
**Pull request by viclim**

### CODE REVIEW

**Comment by calumhalcrow** [comment]
@viclim I guess you might want to rebase this?

File: src/js/actors.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+function receiveArticles(response) {
+  return {
+    type : RECEIVE_ARTICLES,
+    articles: response.data.children.map(child => child.data),
+    receivedAt: Date.now()
+  };
+}
+
+;
```
Comment (calumhalcrow) [inline src/js/actors.js:20]: eslint is ok with this?

**Comment by viclim** [comment]
Should have set the title as WIP earlier.
cc3ca825a8ea24e935912afa25cfceefa20085e49736ff42b2d159ad526d2473

**Comment by viclim** [comment]
Outdated.

**Pull request closed without merging**